### PR TITLE
GROW-168 Google Pay

### PR DESCRIPTION
**Pull request by eddieferrer**
* Adds Google Pay checkout option for Braintree Drop In UI  based on experiment setting.

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -55,6 +55,9 @@ module.exports = {
 			url: 'www.paypal.com',
 			environment: 'production'
 		},
+		googlePay: {
+			merchantId: 'f5swrg87wym8bx37'
```
Comment (mcstover) [inline config/index.js:59]: do we need to blank this out for sandbox?

File: src/components/Checkout/CheckoutDropInPaymentWrapper.vue [hunk 2]
```diff
@@ -53,9 +68,40 @@ export default {
...
+			// applePay & googlePay payment methods experiment - EXP-GROW-168-Aug2020
+			const paymentMethodsExperiment = this.apollo.readFragment({
+				id: 'Experiment:braintree_dropin_apple_google',
+				fragment: experimentVersionFragment,
+			}) || {};
+			if (paymentMethodsExperiment.version === 'shown') {
+				this.paymentTypes = ['paypal', 'card', 'googlePay'];
+			}
+			// Fire Event for EXP-GROW-168-Aug2020
+			if (paymentMethodsExperiment.version && paymentMethodsExperiment.version !== 'unassigned') {
```
Comment (mcstover) [inline src/components/Checkout/CheckoutDropInPaymentWrapper.vue:92]: Once deployed, please verify this is firing correctly. We normally want to reserve analytics events until mounted or after...

File: config/index.js (hunk 1 above)
Comment (eddieferrer) [reply config/index.js:59]: @mcstover according to the docs and my testing, this is fine. It is required for production, but won't hurt anything if it's there for sandbox

File: src/components/Checkout/CheckoutDropInPaymentWrapper.vue (hunk 2 above)
Comment (eddieferrer) [reply src/components/Checkout/CheckoutDropInPaymentWrapper.vue]: @mcstover good catch, fixed this

**Pull request merged**